tasks/linux.sh

```diff
@@ -38,7 +38,7 @@ for p in "${package_managers[@]}"; do
 done
 
 [[ $available_manager ]] || {
-  echo '{ "status": "No package managers found", "message": "Must be one of: [apt, yum]" }'
+  echo '{ "status": "error", "message": "No package managers found", "message": "Must be one of: [apt, yum]" }'
```

Comment:
Hey @m0dular I had envisioned closing this PR in favor of  https://github.com/puppetlabs/puppetlabs-package/pull/117 (which carried over most of your original work), can we continue development on that branch (specifically my fork of this module)?

Reply:
Also note the keys that the error expects, the thing that stands out to me immediately is we expect the key `msg`, not `message`. ;)

Reply:
@donoghuc Oh sure, I had just pushed this change as part of something else I was working on.  I'll go ahead and close it.